## Generalized Enhancer Predictor (GEP) 

Generalized Enhancer Predictor (GEP) is a Random Forest based machine-learning tool for identification of putative enhancers using chromatin dynamics data. The Random Forest model is implemented in python. The package also contains accessary programs required for the analysis.

Documentation can be found at https://generalized-enhancer-predictor-gep.readthedocs.io/en/latest/

## Article:

https://www.biorxiv.org/content/early/2018/09/20/421230

## License:

This project is licensed under the GNU Lesser General Public License v3.0.

## References:
	@article {Jhanwar421230,
		author = {Jhanwar, Shalu and Ossowski, Stephan and Davila-Velderrain, Jose},
		title = {Genome-wide active enhancer identification using cell type-specific signatures of epigenomic activity},
		year = {2018},
		doi = {10.1101/421230},
		publisher = {Cold Spring Harbor Laboratory},
		abstract = {Recently enhancers have emerged as key players regulating crucial mechanisms such as cell fate determination and establishment of spatiotemporal patterns of gene expression during development. Due to their functional and structural complexity, an accurate in silico identification of active enhancers under specific conditions remain challenging. We present a novel machine learning based method that derives epigenomic patterns exclusively from experimentally characterized active enhancers contrasted with a weighted set of non-enhancer genomic regions. We demonstrate better predictive performance over previous methods, as well as wide generalizability by identifying and annotating active enhancers genome-wide across different tissues/cell types in human and mouse.},
		URL = {https://www.biorxiv.org/content/early/2018/09/20/421230},
		eprint = {https://www.biorxiv.org/content/early/2018/09/20/421230.full.pdf},
		journal = {bioRxiv}
	}

## Support:

Please use “GitHub Issues” to report bugs occur during the implementation of the software.
